packages/components/src/Disclosure/docs/inline.tsx

```diff
@@ -0,0 +1,20 @@
+import { Card, Disclosure, DisclosureHeader, DisclosurePanel, Div } from "@hopper-ui/components";
```

Comment:
i understand what you are trying to do here, it's just that looking at the rendered example, it looks like it's exaclty the same as the example from the preview. So from a first glance, it's not clear what this story is. .

Addiotionally, shouldn't this use case be called Variants? or Variant

Reply:
It was variants, but it was changed due to one of Francis' comments. We wanted to highlight the inline variant. The standalone variant is the one that's the same as the preview. didn't make much sense to show it here.
I'd suggest speaking with @fraincs about it.

Reply:
I actually suggested later on to revert to Variant for consistency, in a direct message, this might have been lost in translation.

Reply:
oh sorry. I may have missed it or misunderstood.